index.bs

```diff
@@ -3242,6 +3287,43 @@ The sample code for generating and registering a new key follows:
       });
 </pre>
 
+## Registration with Platform Authenticator ## {#sample-registration-with-platform-authenticator}
+
+This is flow for when the [=[RP]=] is only interested in creating a public key credential with 
+[=platform authenticators=].
+
+1. The user visits example.com. At this point, the user likely already logged in. 
+
+1. The [=[RP]=] script runs the code snippet below. 
+
+1. The client asks the user if s/he is interested in registering a credenital with platform authenticators. 
```

Comment:
suggest:
The user agent asks the user whether they are willing to register with the [=[RP]=] using an available [=platform authenticator=].